src/dialogs/permissions.tsx

```diff
@@ -291,7 +324,7 @@ const EditPermissionsModal = ({ dialogResult, selected, path } : {
                   isInline
                 />}
                 <Form isHorizontal>
-                    {superuser.allowed && accounts && groups &&
+                    {superuser.allowed && accounts && groups && is_user_equal && is_group_equal &&
```

Comment:
OK, so if you have files with different owner/permissions, you can't change their owner/group to all be the same, but you *can* change their permissions and executable flag to all be the same? that feels a bit arbitrary to me.

Not a veto, but it's not intuitive why this is so asymmetric.